definitions/npm/react-native-webview_v5.x.x/flow_v0.88.x-/react-native-webview_v5.x.x.js

```diff
@@ -76,12 +121,11 @@ declare module 'react-native-webview' {
   |};
 
   declare export type WebViewNativeConfig = {|
-    component?: React$ComponentType<WebViewSharedProps>,
+    component?: React$ElementType,
```

Comment:
Are you sure about allowing strings here? Their TS defs seems to only allow components.